src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={cn('', {
+              'is-active': tab.id === selectedTabId,
+            })}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => onTabSelected(tab.id)}
+            >
+              {tab.title}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+
+    <div className="block" data-cy="TabContent">
+      {tabs.map(tab => (
+        <div key={tab.id}>
+          {selectedTabId === tab.id
+            ? <>{tab.content}</>
+            : null}
```

Comment:
It's better to create a separate variable and use it here

`const selectedTab = tabs.find(tab => tab.id === selectedTabid) || tabs[0]`